File: emailTest.js
var nodemailer = require('nodemailer');

var transporter = nodemailer.createTransport({
  service: 'bulkmail.its.csulb.edu ',
  port: 465,
  secure: true
  /*auth: {
    user: '',
    pass: ''
  }*/
});

var mailOptions = {
  from: 'es-records@csulb.edu',
  to: 'ngoalvin@rocketmail.com',
  subject: 'Sending NEW Email using Node.js',
  text: 'That was easy!'
};

transporter.sendMail(mailOptions, function(error, info){
  if (error) {
    console.log(error);
  } else {
    console.log('Email sent: ' + info.response);
  }
});

File: emailTest.py
import smtplib

sender = 'es-records@csulb.edu'
receivers = ['Alvin.Ngo@csulb.edu']

message = """From: es-records <es-records@csulb.edu>
To: Alvin <Alvin.Ngo@csulb.edu>
Subject: SMTP e-mail test

This is a test e-mail message.
"""

try:
   smtpObj = smtplib.SMTP('localhost')
   smtpObj.sendmail(sender, receivers, message)         
   print "Successfully sent email"
except smtplib.SMTPException:
   print "Error: unable to send email"